### PR TITLE
add config.properties set key and values

### PR DESCRIPTION
**Pull request by DariuszCzajkowski**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: pom.xml [hunk 1]
```diff
@@ -68,6 +72,12 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.10.1</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```
Comment (rafalmisiak) [inline pom.xml]: po co Ci junit4 ?

Comment (DariuszCzajkowski) [reply pom.xml]: testy jakieś próbowałem tworzyć. Tak wiem już 5 jest

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**